cli/src/codeowners.rs

```diff
@@ -30,20 +33,35 @@ impl CodeOwners {
             all_locations.find_map(|location| locate_codeowners(&repo_root, location));
 
         codeowners_path.map(|path| {
-            let owners_result = GitHubOwners::from_path(&path);
+            let owners_result = File::open(&path)
+                .map_err(anyhow::Error::from)
+                .and_then(|file| GitHubOwners::from_reader(&file).map(Owners::GitHubOwners))
+                .or_else(|_| {
```

Comment:
This makes sense but should we be doing this off of env vars as we add support for other providers?

Reply:
Totally, I agree. We've got to redo how we use codeowners anyways since as-is it's not really useful. At that point, I think we should start considering finding and parsing codeowners files based on env.